BridgeAppSDK/SBAPermissionsManager.m

```diff
@@ -237,18 +238,29 @@ - (NSString *)permissionDescriptionForType:(SBAPermissionsType)type {
         case SBAPermissionsTypeLocalNotifications:
             return NSLocalizedStringWithDefaultValue(@"SBA_REMINDER_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Allowing notifications enables the app to show you reminders for activity sessions.", @"");
         case SBAPermissionsTypeLocation:
-            return NSLocalizedStringWithDefaultValue(@"SBA_LOCATION_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Using your GPS enables the app to accurately determine distances travelled. Your actual location will never be shared.", @"");
+            return [self bundlePermissionDescriptionForKey: @"NSLocationWhenInUseUsageDescription" defaultValue:@"Using your GPS enables the app to accurately determine distances travelled. Your actual location will never be shared."];
         case SBAPermissionsTypeCoremotion:
-            return NSLocalizedStringWithDefaultValue(@"SBA_COREMOTION_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Using the motion co-processor allows the app to determine your activity, helping the study better understand how activity level may influence disease.", @"");
+            return [self bundlePermissionDescriptionForKey: @"NSMotionUsageDescription" defaultValue:@"Using the motion co-processor allows the app to determine your activity, helping the study better understand how activity level may influence disease."];
         case SBAPermissionsTypeMicrophone:
-            return NSLocalizedStringWithDefaultValue(@"SBA_MICROPHONE_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Access to microphone is required for your Voice Recording Activity.", @"");
-        case SBAPermissionsTypeCamera:
+            return [self bundlePermissionDescriptionForKey: @"NSMicrophoneUsageDescription" defaultValue:@"Access to microphone is required for your Voice Recording Activity."];
         case SBAPermissionsTypePhotoLibrary:
+            return [self bundlePermissionDescriptionForKey: @"NSPhotoLibraryUsageDescription" defaultValue:@"Photo Library"];
+        case SBAPermissionsTypeCamera:
```

Comment:
At least MoleMapper needs access to the camera (though probably gets the permissions outside of Bridge). Does the photo library permission include camera, and we are deprecating a separate camera type? Or...?